orbit_determination/test_ekf.py

```diff
@@ -43,7 +46,7 @@ def imu_init(dt: float) -> IMU:
         sensor_noise_params_accel_y,
         sensor_noise_params_accel_z,
     ]
-
+    # sigma_v [units/sqrt(Hz)] & scale_factor_error [-]
```

Comment:
Sorry, what are the actual units? (deg/s)/sqrt(Hz)? (rad/s)/sqrt(Hz)?

Either way, 5e-10 and 5e-9 are very small. Are these power spectral densities? I believe SensorNoiseParams takes in sqrt of power spectral density.